content/rc/subscriptions/create-fixed-subscription.md

```diff
@@ -10,31 +10,49 @@ aliases: /rv/administration/setup_and_editing/create-subscription/
          /rc/administration/setup-and-editing/create-subscription/
          /rc/administration/setup/create-subscription/
 ---
-Fixed Size subscription plans support low throughput workflows.  Several tiers are available, each designed for different memory sizes and integration requirements.
+Fixed Size subscription plans support low throughput workflows.  Several plans are available, each designed for different memory sizes.
 
-When creating your subscription, you'll need to know which tier to choose.
+When creating your subscription, you'll need to choose the plan that suits your needs.
 
 If you're new to Redis Cloud, the [quick start]({{<relref "/rc/rc-quickstart.md">}}) helps you create an account with a free subscription and an initial database.  You also learn how to connect to your database.
 
+## Fixed plans
 
-## Fixed plan subscription tiers
+Fixed plan pricing scales according to the memory size of the database defined in the subscription.  Additional limits also apply, as shown in the tables below (updated February 2024). 
```

Comment:
@cmilesb can we add a link to the marketing pricing [page](https://redis.com/cloud/pricing/) on the word "pricing"?

Reply:
```suggestion
Fixed plan [pricing](https://redis.com/cloud/pricing/) scales according to the memory size of the database defined in the subscription.  Additional limits also apply, as shown in the tables below (updated February 2024). 
```